lib/still/compiler/incremental/output_to_input_file_registry.ex

```diff
@@ -23,10 +25,17 @@ defmodule Still.Compiler.Incremental.OutputToInputFileRegistry do
   """
   @spec recompile(binary()) :: any()
   def recompile(output_file) do
-    Registry.dispatch(__MODULE__, output_file, fn entries ->
-      for {_pid, input_file} <- entries,
-          do: compile_file(input_file)
-    end)
+    source_files =
+      Registry.lookup(__MODULE__, output_file)
+      |> Enum.map(fn {_pid, input_file} ->
+        compile_file(input_file, run_type: :compile_dev)
+      end)
+
+    if Enum.empty?(source_files) do
+      %SourceFile{input_file: ""}
+    else
+      hd(source_files)
```

Comment:
Why are we only returning one?

Reply:
There's only supposed to be one, but the registry returns an array. We only care about one because we are telling the Registry to compile one file.

Reply:
Can it be more than one? If not, can we do a `case` and return `:error` if there are multiple?

Reply:
I changed it so it kills the app if that happens :P It's never supposed to happen and if it did, something went really bad